### PR TITLE
Disruption details page displays the details of published disruptions

### PR DESCRIPTION
**Pull request by alvak1**
Click the disruption row number from `/dashboard` or `/view-all-disruptions` page. The disruption can be edited and the display is updated if disruption is edited.

### CODE REVIEW

**Comment by alainajaffer** [comment]
cancel changes is showing originally should be Close and Return  instead

File: site/pages/__snapshots__/dashboard.test.tsx.snap [hunk 1]
```diff
@@ -1236,7 +1236,7 @@ exports[`pages > dashboard > should render correctly when there are only live di
                 >
                   <a
                     className="govuk-link"
-                    href="/review-disruption/44"
+                    href="/disruption-detail/44?return=%2Fdashboard"
```
Comment (alainajaffer) [inline site/pages/__snapshots__/dashboard.test.tsx.snap:1239]: why %2F ?

Comment (alvak1) [reply site/pages/__snapshots__/dashboard.test.tsx.snap:1239]: %2F is URI encoded value for forward slash `/`

File: site/pages/disruption-detail/disruption-detail.test.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    publishEndTime: "1400",
+    disruptionStartDate: "15/01/2022",
+    disruptionStartTime: "1200",
+    disruptionEndDate: "17/01/2022",
+    disruptionEndTime: "1400",
+    disruptionNoEndDateTime: "",
+    consequences: previousConsequencesInformation,
+};
+
+const ddCookie: DisruptionDetailCookie = {
```
Comment (alainajaffer) [inline site/pages/disruption-detail/disruption-detail.test.tsx:84]: could we give this a better name

File: site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx [hunk 3]
```diff
@@ -34,8 +35,12 @@ const CreateConsequenceNetwork = (props: CreateConsequenceNetworkProps): ReactEl
 
     const stateUpdater = getStateUpdater(setConsequenceNetworkPageState, pageState);
 
-    const router = useRouter();
-    const queryParams = router.query;
+    const queryParams = useRouter().query;
+    const displayCancelButton =
```
Comment (alainajaffer) [inline site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx:39]: could this not just be condition || condition do we need an if statement

File: site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx [hunk 4]
```diff
@@ -118,6 +119,11 @@ const CreateConsequenceServices = (props: CreateConsequenceServicesProps): React
     }, [selectedService]);
 
     const queryParams = useRouter().query;
+    const displayCancelButton =
+        queryParams["return"]?.includes(REVIEW_DISRUPTION_PAGE_PATH) ||
```
Comment (alainajaffer) [inline site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx:123]: same here

Comment (alvak1) [reply site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx:123]: done

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**